server/controllers/refer/referralsController.ts

```diff
@@ -243,9 +264,11 @@ export default class ReferralsController {
       const referral = await this.referralService.getReferral(req.user.token, req.params.referralId)
       const oasysConfirmed =
         typeof req.body.oasysConfirmed === 'undefined' ? referral.oasysConfirmed : req.body.oasysConfirmed
+      const reason = typeof req.body.reason === 'undefined' ? referral.reason : req.body.reason
```

Comment:
Might make sense to `trim` the reason to prevent any unwanted whitespace?

Reply:
Yeah, and hopefully to stop up accepting a string like `\n\n\n` becing a truthy `reason` (haven't checked what logic you have on the task list to say whether this is completed yet)

Reply:
Yeah, and hopefully to prevent us reading a string like `\n\n\n` as a truthy `reason` (haven't checked what logic you have on the task list to say whether this is completed yet)